src/queries.js

```diff
@@ -117,108 +105,97 @@ export default function queries(allQueries) {
         }
       };
 
-      return connect(connectDeclaration, {
-        pure,
-        // some params for queries cannot be retrieved implicitly from state!
-        // still, we want all the others
-        filterValid: false
-      })(
-        class QueriesWrapper extends React.Component {
-          static contextTypes = QueriesContextTypes;
+      return class QueriesWrapper extends React.Component {
+        static contextTypes = QueriesContextTypes;
 
-          static displayName = displayName;
+        static displayName = displayName;
```

Comment:
why isn't this `static displayName = displayName('queries')(Component);` as done in `CommandWrapper`?